cmd/api-errors.go

```diff
@@ -886,6 +905,12 @@ func toAPIErrorCode(err error) (apiErr APIErrorCode) {
 		apiErr = ErrInvalidSSECustomerParameters
 	case errSSEKeyMismatch:
```

Comment:
You will have to add `errSSEKeyMismatch, crypto.ErrCustomerKeyMD5Mismatch` since you started replacing old code and otherwise clients would receive wrong errors.
Same for other SSE-C errors...